packages/rpc/src/configuration.ts

```diff
@@ -12,6 +12,10 @@ export namespace ConfigurationRpc {
       export const Method = 'v1.configuration.get';
       export type Params = undefined;
       export type Response = {
+        navieConfig: {
+          baseUrl?: string;
+          model?: string;
+        };
```

Comment:
Having it all in the same function seems ok to me. That way you can see everything in one place. I think it should be a pretty flat list of options though, rather than nested into objects. Basically, a lot like CLI options.